bigbluebutton-html5/imports/ui/components/chat/chat-dropdown/component.jsx

```diff
@@ -129,11 +144,14 @@ class ChatDropdown extends PureComponent {
   }
 
   render() {
-    const { intl } = this.props;
+    const {
+      intl,
+      amIModerator,
+    } = this.props;
     const { isSettingOpen } = this.state;
 
     const availableActions = this.getAvailableActions();
-
+    if (!amIModerator && !ENABLE_SAVE_AND_COPY_PUBLIC_CHAT) return null;
```

Comment:
```suggestion
    if (!amIModerator || !ENABLE_SAVE_AND_COPY_PUBLIC_CHAT) return null;
```

Reply:
I think moderators should not be able to save/copy if `enableSaveAndCopyPublicChat: false`

Reply:
They're removed from the array already (https://github.com/bigbluebutton/bigbluebutton/pull/12501/files#diff-185dafe48f96ebd828cc43e28977c48b68538c18d4d4650d34b8fc815a928d9eR101)

Reply:
Tainan pointed out that we still need to proceed with render if Moderator, so we can display the clear chat option